sos/plugins/openstack_ironic.py

```diff
@@ -108,4 +120,23 @@ def setup(self):
             self.add_journal(units="openstack-ironic-discoverd")
             self.add_journal(units="openstack-ironic-discoverd-dnsmasq")
 
+        # ironic-discoverd was renamed to ironic-inspector in Liberty
+        if self.is_installed(self.inspector_package):
```

Comment:
We don't put things behind `is_installed()` unless there are truly harmful effects to not doing it: for one thing, it causes the `all_logs`/`log_size` handling to propagate through the branches and it makes the code ugly and harder to read.

What's the downside here of just running these calls? I can see one journal command, that will collect an error (unit not found), but that's pretty harmless.

Reply:
No downside at all, I was just following the same pattern as above with discoverd. Will remove.

Reply:
OK, thanks - I'll take a look through the OpenStack plugins to see if there are more examples of this pattern - we haven't always been as strict with review as we are now.